test/test_browser.py

```diff
@@ -1977,13 +1976,10 @@ def test_sdl_glshader2(self):
   def test_gl_glteximage(self):
     self.btest('gl_teximage.c', '1', args=['-lGL', '-lSDL'])
 
-  @parameterized({
-    '': ([],),
-    'pthreads': (['-pthread', '-sPROXY_TO_PTHREAD', '-sOFFSCREEN_FRAMEBUFFER'],),
-  })
   @requires_graphics_hardware
-  def test_gl_textures(self, args):
-    self.btest_exit('gl_textures.c', args=['-lGL', '-g', '-sSTACK_SIZE=1MB'] + args)
+  @also_with_proxy_to_pthread
+  def test_gl_textures(self):
+    self.btest_exit('gl_textures.c', args=['-lGL', '-g', '-sSTACK_SIZE=1MB', '-sOFFSCREEN_FRAMEBUFFER'])
```

Comment:
This did not always test with OFFSCREEN_FRAMEBUFFER before. Do we not error on that being used without pthreads?

Reply:
No, I think think its harmlessly ignored.

Reply:
Maybe we should error ?  But we don't today.